### PR TITLE
Refactor/refactor dh filters to use signals

### PR DESCRIPTION
**Pull request by mimse**
https://github.com/Energinet-DataHub/greenforce-frontend/issues/3016

### CODE REVIEW

File: libs/dh/esett/feature-outgoing-messages/src/lib/filters/dh-filters.component.ts [hunk 1]
```diff
@@ -114,17 +119,32 @@ export class DhOutgoingMessagesFiltersComponent implements OnInit {
...
+      actorNumber: dhMakeFormControl(actorNumber),
+      status: dhMakeFormControl(status),
+      period: dhMakeFormControl(period),
+      created: dhMakeFormControl(created),
+      latestDispatch: dhMakeFormControl(latestDispatch),
     });
 
+    this.formReset$
+      .pipe(takeUntilDestroyed(this.destoryRef))
+      .subscribe(() => this.formGroup.reset(this.initial()));
```
Comment (dzhavat) [inline libs/dh/esett/feature-outgoing-messages/src/lib/filters/dh-filters.component.ts:146]: question: is there a need to reset the form this way? Doesn't the reset button do it properly?

Comment (mimse) [reply libs/dh/esett/feature-outgoing-messages/src/lib/filters/dh-filters.component.ts:146]: As fare as I can tell it only resets the search input field, and not the reset

Comment (dzhavat) [reply libs/dh/esett/feature-outgoing-messages/src/lib/filters/dh-filters.component.ts:141]: Hmmm ... just tested it without the `formReset$` and it seems to work. :)

Comment (mimse) [reply libs/dh/esett/feature-outgoing-messages/src/lib/filters/dh-filters.component.ts]: You are right, has been removed.

**Review by dzhavat — APPROVED** [review]

**Pull request merged**